composer.json

```diff
@@ -28,7 +28,7 @@
     "config": {
         "sort-packages": true,
         "platform": {
-            "php": "8.0.99"
+            "php": "8.1"
```

Comment:
8.1.99 should be fine for consistency with other components.

Reply:
updated